.github/workflows/algolia-integrity.yml

```diff
@@ -19,32 +31,28 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+
       - name: Fetch
-        run: git fetch origin master
+        run: git fetch origin ${{ github.base_ref }}
 
       - name: Setup env
         uses: the-guild-org/shared-config/setup@main
         with:
-          nodeVersion: 18
+          nodeVersion: ${{inputs.nodeVersion}}
 
       - name: Algolia generate
-        run: yarn workspace website theguild-nextra-algolia -o ${{ secrets.lockFilePath }} --domain ${{ secrets.domain }} --docsBaseDir ${{ secrets.docsBaseDir }}
-
-      - name: Prettier
-        run: yarn prettier
-
-      - name: Compare
-        run: git diff origin/${{ github.base_ref }}.. -- ${{ secrets.lockFilePath }}
+        run: yarn workspace website theguild-nextra-algolia -o ${{ inputs.lockFilePath }} --domain ${{ inputs.domain }} --docsBaseDir ${{ inputs.docsBaseDir }}
```

Comment:
Is there a way for us to avoid using `yarn` here? 
Some repos might not have a workspace, and some might use `pnpm`...

Reply:
I guess maybe we can ask the user to point us to where the website is (using `inputs`, and then we can just run `theguild-nextra-algolia` directly there?

Reply:
Fixed and tested!